.gitignore

```diff
@@ -115,4 +115,6 @@ dmypy.json
 .pyre/
 
 .vscode/
-.vs/
\ No newline at end of file
+.vs/
+
+corpus/
```

Comment:
Why this change?  It's not mentioned in the PR description.

Reply:
Removed.